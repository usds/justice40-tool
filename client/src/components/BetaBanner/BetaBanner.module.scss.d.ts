declare namespace BetaBannerNamespace {
    export interface IBetaBannerScss {
        betaBannerContainer: string;
        betaBanner:string;
        betaPillIcon:string;
        betaHeading: string;
    }
  }

declare const BetaBannerScssModule: BetaBannerNamespace.IBetaBannerScss & {
    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
    locals: BetaBannerNamespace.IBetaBannerScss;
  };

  export = BetaBannerScssModule;
