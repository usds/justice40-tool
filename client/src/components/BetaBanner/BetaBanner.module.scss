@use '../../styles/design-system.scss' as *;

.betaBannerContainer {
  @include u-bg("gray-cool-2");

  .betaBanner {
    @include u-height(3);
    max-width: 70.25rem; // Needs this exact value to align with GovBanner
    @include u-font("body", "3xs");
    @include u-margin-left(auto);
    @include u-margin-right(auto);
    @include u-padding-top(1);
    padding-bottom: 1.75rem;

    @include at-media-max("desktop") {
      @include u-margin-left(2);
      @include u-margin-right(0);
    }

    @include at-media-max("tablet-lg") {
      @include u-height(6);
    }

  }
}
