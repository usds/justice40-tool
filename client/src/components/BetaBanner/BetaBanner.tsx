import React from 'react';

import * as COMMON_COPY from '../../data/copy/common';
import * as styles from './BetaBanner.module.scss';

const BetaBanner = () => {
  return (
    <div className={styles.betaBannerContainer}>
      <div className={styles.betaBanner}>
        {COMMON_COPY.BETA_BANNER_CONTENT}
      </div>
    </div>
  );
};

export default BetaBanner;
