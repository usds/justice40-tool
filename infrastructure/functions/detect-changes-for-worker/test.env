REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1