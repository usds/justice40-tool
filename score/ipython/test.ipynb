{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
