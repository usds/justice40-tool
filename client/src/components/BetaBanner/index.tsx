import BetaBanner from './BetaBanner';

export default BetaBanner;
