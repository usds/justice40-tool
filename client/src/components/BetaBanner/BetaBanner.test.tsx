import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import BetaBanner from './BetaBanner';

describe('rendering of the BetaBanner', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <BetaBanner />
      </LocalizedComponent>,
  );

  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
