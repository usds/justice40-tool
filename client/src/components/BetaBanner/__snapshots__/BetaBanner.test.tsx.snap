// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the BetaBanner checks if component renders 1`] = `
<DocumentFragment>
  <div>
    <div>
      <strong>
        This tool has been updated.
      </strong>
       The 1.0 version of the tool was released on Nov 22, 2022.
    </div>
  </div>
</DocumentFragment>
`;
